src/ui/public/agg_types/metrics/lib/parent_pipeline_agg_helper.js

```diff
@@ -56,6 +56,15 @@ export const ParentPipelineAggHelperProvider = function (Private) {
           write: parentPipelineAggWritter
         }
       ];
+    },
+    getFormat: function (agg) {
+      let subAgg;
+      if (agg.params.customMetric) {
+        subAgg = agg.params.customMetric;
+      } else {
```

Comment:
When do we hit this code-path? Not a 100% familiar with htis, but don't all parent pipelines require a custom-metric?